jboss/container/java/jvm/bash/artifacts/opt/jboss/container/java/jvm/java-default-options

```diff
@@ -92,7 +92,7 @@ initial_memory() {
   fi
 
   # Check for the 'real memory size' and calculate ms from a ratio
-  # given (default is 100%)
+  # given (default is 25%)
```

Comment:
L13 in this file also incorrectly says the default for this is 100

Reply:
@bstansberry I've just updated the line 13, thanks